packages/node/src/lib/environments/environments.ts

```diff
@@ -0,0 +1,37 @@
+import { WithHttp } from '../novu.interface';
```

Comment:
```suggestion
import axios from 'axios';
import { WithHttp } from '../novu.interface';
```
Even if not used, I believe it's required in order to work with `WithHttp`